packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -81,12 +81,21 @@ class SnackBarAction extends StatefulWidget {
   /// The [label] and [onPressed] arguments must be non-null.
   const SnackBarAction({
     Key key,
+    this.labelColor,
```

Comment:
I would call these textColor and disabledTextColor to match what we're passing them too